query/src/storages/fuse/operations/truncate_histor.rs

```diff
@@ -0,0 +1,116 @@
+use std::collections::HashSet;
```

Comment:
1. file name: truncate_histor.rs -> truncate_history.rs 
2. The licenese header is not on top ~

Reply:
oops, got it!